.github/workflows/gradle.yml

```diff
@@ -33,10 +34,12 @@ jobs:
         with:
           java-version: ${{ matrix.java }}
       - name: Optional setup step
+        env:
+          K8S_VERSION: ${{ matrix.k8s }}        
         run: |
           [ -f ./setup.sh ] && ./setup.sh || true
       - name: Build with Gradle
-        run: ./gradlew dependencyUpdates check --parallel --continue
```

Comment:
parallel won't work here, as majority of the specs are using same namespace. We agreed weeks ago to remove the parallel and in some of the next prs, every spec will create it's own namespace.